src/hcf-release/jobs/uaa-create-user/templates/run.erb

```diff
@@ -32,9 +43,13 @@ function retry () {
     shift 2
 
     while test ${i} -lt ${max} ; do
+	printf "Trying: %s\n" "$*"
         if "$@" ; then
+	    status ' SUCCESS'
             break
         fi
+	trouble '  FAILED'
+	status "Waiting ${delay} ..."
```

Comment:
In my testing it actually looked quite nice, and not too cluttered, as I had feared.
So, no, for me the output was not an issue.

Ok. I will remove the urls from the status messages, and leave that to the printed command.
I keep printing the command.